### PR TITLE
chore: implement featured asset styles

### LINKED ISSUE #333: Implement Featured Asset style
Some applications have featured asset which must be styled. Currently we have:

* https://main--moleculardevices--hlxsites.hlx.page/applications/3d-cell-models/organoids/brain-organoids
* https://main--moleculardevices--hlxsites.hlx.page/applications/3d-cell-models/organoids/intestinal-organoids
* https://main--moleculardevices--hlxsites.hlx.page/applications/3d-cell-models/organoids/pulmonary-lung-organoids

To be styled:
* background asset should be from CSS
* margins
* heading color
* text color
* primary & secondary button color

### PR DESCRIPTION
**Pull request by mhaack**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #333 

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.pageapplications/3d-cell-models/organoids/brain-organoids
- After: https://featured-asset--moleculardevices--hlxsites.hlx.page/applications/3d-cell-models/organoids/brain-organoids

- Sample 2: https://featured-asset--moleculardevices--hlxsites.hlx.page/applications/3d-cell-models/organoids/intestinal-organoids
- Sample 3: https://featured-asset--moleculardevices--hlxsites.hlx.page/applications/3d-cell-models/organoids/pulmonary-lung-organoids

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -847,6 +846,58 @@ main .section.wave .default-content-wrapper p:first-of-type {
...
+
+main .section.featured-asset h2 {
+  margin-top: 0;
+  font-family: var(--ff-proxima-regular);
+  font-size: 36px;
+  color: var(--text-green);
+}
+
+main .section.featured-asset h3 {
+  font-size: 25px !important;
```
Comment (duynguyen) [inline styles/styles.css:879]: can we use a more specific selector not to use important here?

Comment (mhaack) [reply styles/styles.css:879]: Trying. The columns block has `.section.columns-container .columns.columns-2-cols > div h3` setting the h3 font, which is super wide actually. Hm let me check if I can change that.

Comment (mhaack) [reply styles/styles.css]: Fixed and also checked https://featured-asset--moleculardevices--hlxsites.hlx.page/about-us#customers-we-serve to look same as before.

**Review by duynguyen — APPROVED** [review]
LGTM thanks

**Pull request merged**